src/components/Reports/PartnerGivingAnalysisReport/PartnerGivingAnalysisReport.tsx

```diff
@@ -61,8 +58,12 @@ export const PartnerGivingAnalysisReport: React.FC<Props> = ({
     [activeFilters, searchTerm],
   );
 
-  const { data, previousData, loading } = usePartnerGivingAnalysisQuery({
-    variables: {
+  // pageSize is intentionally omitted from the dependencies array so that the query isn't reloaded when the page size changes
+  // If all the pages have loaded and the user changes the page size, there's no reason to reload all the pages
+  const pageSize =
```

Comment:
praise: Great find with using the `apiRef` to get the page size! I like how we don't have to drill as many props down!